spring-cloud-starter-single-step-batch-job/pom.xml

```diff
@@ -41,6 +47,40 @@
 			<artifactId>h2</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.amqp</groupId>
+			<artifactId>spring-amqp</artifactId>
```

Comment:
Dependencies like this should be optional so we don't bring in the whole world...